addons/account/models/account_move.py

```diff
@@ -1983,6 +1983,13 @@ def _synchronize_business_models(self, changed_fields):
     # DYNAMIC LINES
     # -------------------------------------------------------------------------
 
+    def get_discount_allocation_account(self):
```

Comment:
Doesn't need to be public
```suggestion
    def _get_discount_allocation_account(self):
```

Reply:
I wouldn't put it in `# DYNAMIC LINES`. `# HELPER METHODS` would be better IMO